src/_companies/europe/sweden/stockholm/qasa/index.md

```diff
@@ -0,0 +1,24 @@
+---
+name:  "Qasa"
+
+continent: "Europe"
+country: "Sweden"
+region: "Stockholm"
+city: "Stockholm"
+
+website: https://qasa.se/
+careers_page: https://careers.qasa.se
+
+work_arrangements:
+  - hybrid
+  - onsite
+
+stack:
+  - "rails"
+  - "postgres"
+  - "react"
+
+created_at: 2023-03-27 01:33:70
+---
+
+At Qasa, we believe rental homes are crucial in empowering mobility and enabling individual and societal growth. Our mission is to re-imagine home rentals to create a happier rental experience for both homeowners and tenants. So more people can spend less time finding a place and more time feeling at home.
```

Comment:
Could you change this to be in the third person? It's the default for all other company descriptions :)

Reply:
Updated in [27d19a5](https://github.com/CalvinWalzel/ruby-companies/pull/20/commits/27d19a58492364f2fc20c14773a5a2e78ecac815)